airbyte-integrations/connectors/source-zoom/source_zoom/source.py

```diff
@@ -15,4 +15,106 @@
 # Declarative Source
 class SourceZoom(YamlDeclarativeSource):
     def __init__(self):
-        super().__init__(**{"path_to_yaml": "zoom.yaml"})
+        super().__init__(**{"path_to_yaml": "manifest.yaml"})
+
+
+
+
+
+x=["airbyte-integrations/connectors/source-activecampaign/source_activecampaign/source.py",
```

Comment:
This seems odd. Why is this part of the zoom source?

Reply:
fixed